app/gui2/src/components/visualizations/JSONVisualization/JsonObjectWidget.vue

```diff
@@ -74,4 +79,8 @@ function entryTitle(key: string) {
 .key {
   color: darkred;
 }
+.key-link {
+  color: blue;
+  text-decoration: underline;
+}
```

Comment:
Only one class is needed for `key`; different styles can be specified for `.key` and `.viewonly .key`.